app/build.gradle

```diff
@@ -1,16 +1,16 @@
 plugins {
```

Comment:
Could you rebase and integrate the kts changes?

Reply:
Whoops, wrong branch -.-